zulipterminal/cli/run.py

```diff
@@ -108,7 +108,7 @@ def parse_zuliprc(zuliprc_str: str) -> Dict[str, Any]:
     while not path.exists(zuliprc_path):
         try:
             fetch_zuliprc(zuliprc_path)
-        except OSError:
+        except (OSError, ValueError):
```

Comment:
I would add the comment you added later in the PR in this commit instead; it is regarding `ValueError` only, so adding the comment and the exception at the same time makes sense.

Reply:
Fixed